### PR TITLE
Scans: SSTI

### PR DESCRIPTION
**Pull request by psiinon**
I'll submit a PR for automating this but it might well take longer to get right than the others.
I'll test it on a personal repo as I expect it will take quite a few attemots to get right ;)

Screenshot 2022-03-31 at 17-04-10 OWASP ZAP – ZAP vs Websites Vulnerable to SSTI

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Comment by kingthorin** [comment]
Can we assume the main plan is to add a first column which is SSTI and passes all of them?

**Comment by psiinon** [comment]
I'm testing the 2 new rules right now. So far they dont both find them all but 90035 (the non blind one) does a v good job.
So yes, once those 2 rules have been published then I'll add them to the results :)

File: site/content/docs/scans/ssti.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+title: "ZAP vs Websites Vulnerable to SSTI"
+type: page
+EditableContent: true
+---
+Websites Vulnerable to SSTI is a set of simple servers which are vulnerable to Server Side Template Injection. 
+
+It is not available online, the GitHub repo is https://github.com/DiogoMRSilva/websitesVulnerableToSSTI
+It is actively maintained by a ZAP contributor: [Diogo Silva](https://github.com/DiogoMRSilva). 
+
+The vulnerabilities are reported by various ZAP scan rules - if any of then find a vulnerability then we count that as a pass.
```
Comment (kingthorin) [inline site/content/docs/scans/ssti.md:11]:
```suggestion
The vulnerabilities are reported by various ZAP scan rules - if any of them find a vulnerability then we count that as a pass.
```

Comment (psiinon) [reply site/content/docs/scans/ssti.md]: Fixed :)

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Pull request merged**